examples/plot_auto_repair.py

```diff
@@ -93,13 +93,17 @@
 # determine the optimal values :math:`\rho^{*}` and :math:`\kappa^{*}`
 
 
-epochs.ch_names
 ar = LocalAutoRejectCV(n_interpolates, consensus_percs, picks=picks,
                        thresh_func=thresh_func)
-epochs_clean = ar.fit_transform(epochs['Auditory/Left'])
 
-evoked = epochs.average()
+# we can fit on one part of the epochs
+ar.fit(epochs['Auditory/Left'])
```

Comment:
Tests are fixed now.

But something is still off here. Autoreject seems to be modifying something inplace. Because the bad channel doesn't quite look right ...

On master, we have:

![master](https://user-images.githubusercontent.com/15852194/29250377-2a5c7ebe-805f-11e7-8258-81e6e97664f2.png)


and on this branch, we have

![branch](https://user-images.githubusercontent.com/15852194/29250356-c97cd170-805e-11e7-873d-7d497a8de579.png)